docs/Makefile

```diff
@@ -2,7 +2,7 @@
 #
 
 # You can set these variables from the command line.
-SPHINXOPTS    =
+SPHINXOPTS    = -W --keep-going #-n
```

Comment:
this is needed and was missed in #1257 . The `-n` will be useful later when we need to check for broken links (atm, this is useless since we have too many unlinked `tensor` calls in the docs).

Reply:
Filed #1259 for this.